redis-image/src/main/docker/run.sh

```diff
@@ -6,7 +6,7 @@
 
 function findmaster() {
   #First check with the sentinels
-  master=$(redis-cli -h ${REDIS_SENTINEL_SERVICE_HOST} -p ${REDIS_SENTINEL_SERVICE_PORT} --csv SENTINEL get-master-addr-by-name mymaster | tr ',' ' ' | cut -d' ' -f1)
+  master=$(timeout -t 3 redis-cli -h ${REDIS_SENTINEL_SERVICE_HOST} -p ${REDIS_SENTINEL_SERVICE_PORT} --csv SENTINEL get-master-addr-by-name mymaster | tr ',' ' ' | cut -d' ' -f1)
```

Comment:
The default timeout of 30 seconds slows down initial launch by quite a bit.  I added a fast timeout of 3 seconds when asking the sentinels who the current master is.